### PR TITLE
port to Gtk+3

### PR DESCRIPTION
**Pull request by yell0wfl4sh**
@quozl  please review!

### CODE REVIEW

File: readsdcomics.py [hunk 1]
```diff
@@ -65,47 +68,47 @@ def __init__(self, handle):
         self.zoom_image_to_fit = True
         self.total_pages = 0
 
-        self.connect("expose_event", self.area_expose_cb)
+        self.connect("draw", self.area_expose_cb)
```
Comment (quozl) [inline readsdcomics.py]: Change also the name of the callback and the arguments.

File: readsdcomics.py [hunk 2]
```diff
@@ -120,9 +123,9 @@ def __init__(self, handle):
         self.saved_screen_width = 0
         self.eventbox.grab_focus()
         
-        pixmap = gtk.gdk.Pixmap(None, 1, 1, 1)
-        color = gtk.gdk.Color()
-        self.hidden_cursor = gtk.gdk.Cursor(pixmap, pixmap, color, color, 0, 0)
+        # pixmap = Gdk.Pixmap(None, 1, 1, 1)
+        # color = Gdk.Color()
+        # self.hidden_cursor = Gdk.Cursor.new(pixmap, pixmap, color, color, 0, 0)
```
Comment (quozl) [inline readsdcomics.py]: Why is this commented out?

File: readsdcomics.py [hunk 3]
```diff
@@ -243,13 +246,13 @@ def create_new_toolbar(self):
...
+        total_page_item = Gtk.ToolItem()
+        self.total_page_label = Gtk.Label()
 
-        label_attributes = pango.AttrList()
-        label_attributes.insert(pango.AttrSize(14000, 0, -1))
-        label_attributes.insert(pango.AttrForeground(65535, 65535, 
-                                                     65535, 0, -1))
+        label_attributes = Pango.AttrList()
+        # label_attributes.insert(Pango.AttrSize(14000, 0, -1))
+        # label_attributes.insert(Pango.AttrForeground(65535, 65535,
```
Comment (quozl) [inline readsdcomics.py]: Why commented out?

File: readsdcomics.py [hunk 4]
```diff
@@ -476,8 +479,8 @@ def next_page(self):
         self.page = page
 
     def area_expose_cb(self, area, event):
```
Comment (quozl) [inline readsdcomics.py]: Change function name, arguments, and see if the new arguments can be used.

**Review by quozl — COMMENTED** [review]
Thanks, interesting.

**Comment by yell0wfl4sh** [comment]
Updated changes, not sure about the removal of label_attributes, I wasn't able to find it's GTK+3 alternative and also didn't notice much difference after removing it, so decided to delete it. please review!

**Pull request merged**